shared/oae/api/oae.api.group.js

```diff
@@ -125,6 +125,58 @@ define(['exports', 'jquery', 'underscore', 'oae.api.util'], function(exports, $,
     };
 
     /**
+     * Get all the invitations for a group
+     *
+     * @param  {String}          groupId                        Id of the group we're trying to retrieve the invitations for
+     * @param  {Function}        callback                       Standard callback function
+     * @param  {Object}          callback.err                   Error object containing error code and error message
+     * @param  {Object}          callback.result                Response object containing the group invitations
+     * @param  {Invitation[]}    callback.result.results        Every invitation associated to the group
```

Comment:
I feel like `Get all the invitations...` in the summary and `Every invitation associated...` in the result is re-iterated enough